rdkcentral/Dobby
# Fixed bug causing plugin launcher test to freeze.

### Description
Test was waiting for pid of container, which normally is provided in stdin by crun. In this case crun was not providing it which caused infinite wait. Now test provides this pid field itself.

### Test Procedure
Go to ~/srcDobby/tests/test_runner/
run "python3 plugin_launcher.py -p1|

Without this patch this command will freeze forever. With this patch there should be "All 1 succeed" message.

### Type of Change
- Bug fix

### Requires Bitbake Recipe changes?
- none
